streams-bootstrap-core/src/test/java/com/bakdata/kafka/ConsumedXTest.java

```diff
@@ -273,7 +274,53 @@ void shouldUseOffsetResetPolicy() {
             @Override
             public void buildTopology(final StreamsBuilderX builder) {
                 final KStreamX<String, String> input =
-                        builder.stream("input", ConsumedX.with(AutoOffsetReset.LATEST));
+                        builder.stream("input", ConsumedX.with(AutoOffsetReset.latest()));
+                input.to("output");
+            }
+        };
+        try (final KafkaContainer kafkaCluster = KafkaTest.newCluster()) {
+            kafkaCluster.start();
+            final RuntimeConfiguration configuration = RuntimeConfiguration.create(kafkaCluster.getBootstrapServers())
+                    .withNoStateStoreCaching()
+                    .withSessionTimeout(SESSION_TIMEOUT);
+            final KafkaTestClient testClient = new KafkaTestClient(configuration);
+            testClient.createTopic("input");
+            testClient.createTopic("output");
+            testClient.send()
+                    .with(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class)
+                    .with(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class)
+                    .to("input", List.of(new SimpleProducerRecord<>("foo", "bar")));
```

Comment:
Does it make sense to extract some helper function for this configuration? The same is repeated below in this file and similarly done in other places as well.